gnomad/utils/annotations.py

```diff
@@ -32,6 +33,33 @@
     "pab_max": (0, 1, 50),
 }
 
+VRS_CHROM_IDS = {
```

Comment:
will these be the same for both build 37 and 38?

Reply:
I just realized that I do not know the answer to this, I'll reach out to their team about it

Reply:
it turns out they will not be the same , so I'll have to make a separate one on GRCh37 looks like . Should be simple enough